Contest_API.md

```diff
@@ -760,6 +762,18 @@ can only be non-null when start time is null.
 The request should fail with a 403 error code if the contest is started or within 30s of
 starting, or if the new start time is in the past or within 30s.
 
+Clients with the `contest_thaw` [capability](#capabilities) have the ability to
+set a time when the contest will be thawed via a PATCH method.
+
+The Patch must include a valid JSON object with exactly three properties:
+the contest `id` (used for verification) and a `contest_thaw_time`, which can
+either be a `<TIME>` value or the specialstring value `"now"`, which indicates
+the contest should be thawed at the current time.
+
+The request should fail with a 4xx error code if the contest can't be thawed at the given
```

Comment:
Shouldn't this be 403 like the start time?

Reply:
I swear I saw 4xx but I was just blind.

Reply:
I've tried that explanation but it works better when you do it. 😂